test/infer/test_reparam.py

```diff
@@ -315,7 +328,7 @@ def get_expected_probe(loc, concentration):
 
     def get_actual_probe(loc, concentration):
         kernel = NUTS(model_act)
-        mcmc = MCMC(kernel, num_warmup=1000, num_samples=10000, num_chains=1)
+        mcmc = MCMC(kernel, num_warmup=1000, num_samples=20000, num_chains=1)
```

Comment:
It seems that this test is flaky (sometimes failed in CI) so increases this number to make it less flaky.